gnovm/cmd/gno/list.go

```diff
@@ -0,0 +1,129 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"flag"
+	"fmt"
+	"io"
+	"text/template"
+
+	"github.com/gnolang/gno/gnovm/pkg/packages"
+	"github.com/gnolang/gno/tm2/pkg/commands"
+)
+
+type listCfg struct {
+	json   bool
+	deps   bool
+	test   bool
+	format string
+}
+
+func newListCmd(io commands.IO) *commands.Command {
+	cfg := &listCfg{}
+	return commands.NewCommand(
+		commands.Metadata{
+			Name:       "list",
+			ShortUsage: "gno list [flags] <pattern> [patterns...]",
+			ShortHelp:  "lists the named packages",
+			LongHelp:   "List lists the named packages, one per line.",
+		},
+		cfg,
+		func(_ context.Context, args []string) error {
+			return execList(cfg, args, io)
+		})
+}
+
+func (c *listCfg) RegisterFlags(fs *flag.FlagSet) {
+	fs.BoolVar(&c.json, "json", false, "Output in JSON")
+	fs.BoolVar(&c.deps, "deps", false, "Load dependencies")
+	fs.BoolVar(&c.test, "test", false, "Load tests")
+	fs.StringVar(&c.format, "f", "", "Output template in go-template format")
```

Comment:
If it's template output, using `-format` would look better. `-f` could be interpreted as force.

Reply:
I see. then no need to change the flag :+1